AdminWebsite/AdminWebsite/Services/AppRoleService.cs

```diff
@@ -63,38 +64,27 @@ public async Task<List<Claim>> GetClaimsForUserAsync(string uniqueId, string use
             return claims;
         }
 
-        private static List<Claim> MapUserRoleToAppRole(int userUserRoleId)
+        private static List<Claim> MapUserRoleToAppRole(List<JusticeUserRole> userRoles)
         {
-            var appRole = userUserRoleId switch
-            {
-                (int) JusticeUserRole.CaseAdmin => AppRoles.CaseAdminRole,
-                (int) JusticeUserRole.Vho => AppRoles.VhOfficerRole,
-                (int) JusticeUserRole.Judge => AppRoles.JudgeRole,
-                (int) JusticeUserRole.StaffMember => AppRoles.StaffMember,
-                (int) JusticeUserRole.VhTeamLead => AppRoles.VhOfficerRole,
-                _ => null
-            };
-
             var claims = new List<Claim>();
-            if (appRole != null)
+            foreach (JusticeUserRole role in userRoles)
             {
-                claims.Add(new Claim(ClaimTypes.Role, appRole));
+                var appRole = role switch
+                {
+                    JusticeUserRole.CaseAdmin => AppRoles.CaseAdminRole,
+                    JusticeUserRole.Vho => AppRoles.VhOfficerRole,
+                    JusticeUserRole.Judge => AppRoles.JudgeRole,
+                    JusticeUserRole.StaffMember => AppRoles.StaffMember,
+                    JusticeUserRole.VhTeamLead => AppRoles.VhOfficerRole,
+                    _ => null
+                };
+                if (appRole != null)
+                {
+                    claims.Add(new Claim(ClaimTypes.Role, appRole));
+                }
             }
-
+            
             return claims;
         }
-
-        public enum JusticeUserRole
```

Comment:
This was an intentional enum to avoid tightly coupling this to the Bookings API client.

Reply:
I think this must go to avoid code maintenance in 2 different places for the same enum